plugins/dell-k2/fu-dell-k2-plugin.c

```diff
@@ -415,6 +415,17 @@ fu_dell_k2_plugin_prepare(FuPlugin *plugin,
 		fu_device_add_private_flag(device, FU_DEVICE_PRIVATE_FLAG_SKIPS_RESTART);
 	}
 
+	/* register post message */
+	if (fu_plugin_get_config_value_boolean(plugin, FWUPD_DELL_K2_PLUGIN_CONFIG_UOD)) {
+		g_autoptr(FwupdRequest) request = fwupd_request_new();
+
+		fu_device_add_request_flag(device, FWUPD_REQUEST_FLAG_ALLOW_GENERIC_MESSAGE);
+		fwupd_request_set_kind(request, FWUPD_REQUEST_KIND_POST);
+		fwupd_request_set_id(request, FWUPD_REQUEST_ID_REMOVE_USB_CABLE);
+		fwupd_request_add_flag(request, FWUPD_REQUEST_FLAG_ALLOW_GENERIC_MESSAGE);
+		return fu_device_emit_request(device, request, progress, error);
```

Comment:
This results in multiple pop-up dialogs that is anooying, will change to pick only the first device to register.